src/node-lib/curriculum-api-2023/helpers/index.ts

```diff
@@ -0,0 +1,25 @@
+import { TierSchema } from "../queries/unitListing/unitListing.schema";
+
+export const toSentenceCase = (str: string) => {
+  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+};
+
+const hasCoreTierSet = new Set(["maths-ks4"]);
+
+export const filterOutCoreTier = (
```

Comment:
can we add a comment explaining why we need to do this

Reply:
✅